pkg/webmiddleware/cookie.go

```diff
@@ -0,0 +1,45 @@
+// Copyright © 2020 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package webmiddleware
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/gorilla/securecookie"
+)
+
+type ctxKeyType struct{}
+
+var ctxKey ctxKeyType
```

Comment:
These context keys are not for all of `pkg/webmiddleware`, so better to rename them to `secureCookieCtxKey` or something.